VMBackup/main/handle.py

```diff
@@ -564,6 +565,17 @@ def enable():
         error_msg = 'Failed to call the daemon'
         exit_with_commit_log(temp_status, temp_result,error_msg, para_parser)
 
+def set_file_permission(file_path, permission):
+    os.system('chmod ' + permission + " " + file_path)
```

Comment:
setting permission at every enable is not recommended. Change permission only if it is not 0644